pkgs/by-name/ja/jackbox-utility/package.nix

```diff
@@ -0,0 +1,52 @@
+{ lib
+, stdenvNoCC
+, fetchurl
+, unzip
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "jackbox-utility";
+  version = "1.3.7+1";
+
+  src = fetchurl {
+    url = "https://github.com/JackboxUtility/JackboxUtility/releases/download/${version}/JackboxUtility_Linux.zip";
+    hash = "sha256-fPLcbVqZ5rXkYRmBsOacwuJaIK47/JZMqsJRQTTsZRM=";
+  };
+
+  dontUnpack = true;
+
+  nativeBuildInputs = [ unzip ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin $out/share/applications
+    unzip -d $out/bin $src
+    mv $out/bin/JackboxUtility $out/bin/${pname}
```

Comment:
Please do not reference `pname` unnecessarily.

Reply:
Thought it would be better to reference than to repeat it, but makes sense